Admin/AbstractAdmin.php

```diff
@@ -540,6 +538,15 @@
     private $breadcrumbsBuilder;
 
     /**
+     * @var array
+     *
+     * @deprecated since 3.x, to be removed in 4.0
+     * NEXT_MAJOR: remove this property, the __get, __set, __isset, __unset magic methods and the
+     * triggerPropertyDeprecation method.
+     */
+    private $cacheIsGranted = array();
```

Comment:
Technically, yes. But too many protection around this property can create BC breaks when all those stuff is removed. See https://github.com/symfony/symfony/pull/20769#issuecomment-265087953.